android/app/src/main/java/org/fossasia/openevent/common/api/OpenEventAPI.java

```diff
@@ -46,24 +47,45 @@
     @GET("speakers?include=sessions&fields[session]=title")
     Call<List<Speaker>> getSpeakers();
 
+    @GET("speakers?include=sessions&fields[session]=title")
+    Single<List<Speaker>> getSpeakersSingle();
+
     @GET("sponsors")
     Call<List<Sponsor>> getSponsors();
 
+    @GET("sponsors")
+    Single<List<Sponsor>> getSponsorsSingle();
+
     @GET("sessions?include=microlocation,track&fields[microlocation]=name&fields[track]=name")
     Call<List<Session>> getSessions();
 
+    @GET("sessions?include=microlocation,track&fields[microlocation]=name&fields[track]=name")
+    Single<List<Session>> getSessionsSingle();
+
     @GET("../{id}?include=social_links,speakers_call,event_copyright")
     Call<Event> getEvent(@Path("id") int eventId);
 
+    @GET("../{id}?include=social_links,speakers_call,event_copyright")
+    Single<Event> getEventSingle(@Path("id") int eventId);
+
     @GET("microlocations")
     Call<List<Microlocation>> getMicrolocations();
 
+    @GET("microlocations")
+    Single<List<Microlocation>> getMicrolocationsSingle();
+
     @GET("tracks?include=sessions&fields[session]=title")
     Call<List<Track>> getTracks();
 
+    @GET("tracks?include=sessions&fields[session]=title")
+    Single<List<Track>> getTracksSingle();
+
     @GET("session-types")
     Call<List<SessionType>> getSessionTypes();
 
+    @GET("session-types")
+    Single<List<SessionType>> getSessionTypesSingle();
+
```

Comment:
Rather than using single, use Observable. I know single is more specific but observable is more general